warehouse/templates/accounts/login.html

```diff
@@ -11,16 +11,28 @@
     {% endif %}
 
     {% if form.errors.__all__ %}
-    <ul class="errors">{% for error in form.errors.__all__ %}<li>{{ error }}</li>{% endfor %}</ul>
+    <ul class="errors">
+      {% for error in form.errors.__all__ %}
+      <li>{{ error }}
```

Comment:
Hi @sigmavirus24, thanks for the feedback. The end tag can be omitted here per the HTML5 spec, see https://html.spec.whatwg.org/multipage/semantics.html#the-li-element